fastlane_core/spec/core_ext/shellwords_ext_spec.rb

```diff
@@ -236,7 +236,7 @@ def simulate_windows_shell_unwrapping(string)
   regex = /^("|')(([^"])(\S*)([^"]))("|')$/
   unless string.to_s.match(regex).nil?
     string = string.to_s.match(regex)[2] # get only part in quotes
-    string.to_s.gsub!('""', '"') # remove double double quotes
+    string.to_s.gsub!('""', '"') # remove doubled double quotes
```

Comment:
this makes the no doubled words rule happy and coincidentally decreases ambiguity/makes it easier to read/understand.